dist/ramda.js

```diff
@@ -3263,6 +3246,35 @@
 
     /**
      * Converts an object into an array of key, value arrays.
+     * Only the object's own properties are used.
+     * Note that the order of the output array is not guaranteed to be
+     * consistent across different JS platforms.
+     *
+     * @func
+     * @memberOf R
+     * @category Object
+     * @sig {String: *} -> [[String,*]]
+     * @param {Object} obj The object to extract from
+     * @return {Array} An array of key, value arrays from the object's own properties.
+     * @example
+     *
+     *      R.toPairs({a: 1, b: 2, c: 3}); //=> [['a', 1], ['b', 2], ['c', 3]]
+     */
+    var toPairs = _curry1(function toPairs(obj) {
+        var pairs = [];
+        for (var prop in obj) {
+            if (Object.prototype.hasOwnProperty.call(obj, prop)) {
```

Comment:
I meant to add that if people want stand-alone implementations, a library like [cherries][ch] might be a good fit.  I haven't really played with it much.  But it's goals were to provide simple individual functional constructs with no dependencies at all.  I guess we'd have to ask @casperin if there are any plans to break it into more easily digestible modules for NPM, but I never thought about such separability being an important goal for Ramda.


  [ch]: https://github.com/casperin/cherries/blob/master/cherries.js